tests/test-edit-flow-notifications.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+class WP_Test_Edit_Flow_Notifications extends WP_UnitTestCase {
+
+	protected static $admin_user_id;
+	protected static $EF_Notifications;
+		
+	public static function wpSetUpBeforeClass( $factory ) {
```

Comment:
Any reason not to call the parent function as well?

```suggestion
	public static function wpSetUpBeforeClass( $factory ) {
		parent::wpSetUpBeforeClass();
```

Reply:
Same thing here, does this have a parent function?

It throws an exception when running PHPUnit: `Call to undefined method WP_UnitTestCase::wpSetUpBeforeClass()`

Reply:
Huh, weird, I guess not!